yarn-project/yarn.lock

```diff
@@ -4167,12 +4167,12 @@ __metadata:
 
 "@noir-lang/noir_js@file:../noir/packages/noir_js::locator=%40aztec%2Faztec3-packages%40workspace%3A.":
   version: 1.0.0-beta.3
-  resolution: "@noir-lang/noir_js@file:../noir/packages/noir_js#../noir/packages/noir_js::hash=73647e&locator=%40aztec%2Faztec3-packages%40workspace%3A."
+  resolution: "@noir-lang/noir_js@file:../noir/packages/noir_js#../noir/packages/noir_js::hash=a967a9&locator=%40aztec%2Faztec3-packages%40workspace%3A."
   dependencies:
     "@noir-lang/acvm_js": "npm:1.0.0-beta.3"
     "@noir-lang/noirc_abi": "npm:1.0.0-beta.3"
     "@noir-lang/types": "npm:1.0.0-beta.3"
-  checksum: 10/4e756f69e5c8fcfe4f1243f026e37d68584852ad18887414ccc74de928ff2c019ea785ae06e48c523744676ccff2993b4d3ed8b971d6478e431af38574b4e924
+  checksum: 10/e544b07c369f63eb6513f11b3e4f203a2c4fd8cfca4965cecd576affc06ceee8c4ec817d220dcd24f8e31c896fc86eb3343f2a49744082c81808336567ac4882
```

Comment:
Not sure why this change got triggered here - I just bootstrapped and it happened